packages/convex-helpers/server/zod.ts

```diff
@@ -637,22 +641,26 @@ type ConvexValidatorFromZod<Z extends z.ZodTypeAny> =
                                                 ],
                                                 "required",
                                                 ConvexValidatorFromZod<Inner>["fieldPaths"]
-                                            >
-                                            // Swap nullable(optional(foo)) for optional(nullable(foo))
-                                            : ConvexValidatorFromZod<Inner> extends Validator<infer T, "optional", infer F>
-                                            ?
-                                            VUnion<
-                                                | null
-                                                | Exclude<ConvexValidatorFromZod<Inner>["type"], undefined>,
-                                                [
-                                                  Validator<T, "required", F>,
-                                                  VNull,
-                                                ],
-                                                "optional",
-                                                ConvexValidatorFromZod<Inner>["fieldPaths"]
-                                            >
-
-                                            : never
+                                              >
+                                            : // Swap nullable(optional(foo)) for optional(nullable(foo))
+                                              ConvexValidatorFromZod<Inner> extends Validator<
+                                                  infer T,
+                                                  "optional",
+                                                  infer F
+                                                >
+                                              ? VUnion<
+                                                  null | Exclude<
+                                                    ConvexValidatorFromZod<Inner>["type"],
+                                                    undefined
+                                                  >,
+                                                  [
+                                                    Validator<T, "required", F>,
+                                                    VNull,
+                                                  ],
+                                                  "optional",
+                                                  ConvexValidatorFromZod<Inner>["fieldPaths"]
+                                                >
+                                              : never
```

Comment:
I wonder why prettier has changed - I often get reformatting on older files - maybe it pre-dated the prettierrc. 
Is there a better practice like pinning the prettier version too?